src/configure/helper/devOps/serviceConnectionHelper.ts

```diff
@@ -68,7 +68,7 @@ export class ServiceConnectionHelper {
                 break;
             }
 
-            if (!(retryCount < 20)) {
+            if (!(retryCount < 40)) {
```

Comment:
GitHub Service endpoint is created synchronously. There shouldn't be retries required

Reply:
After creation we check if endpoint status is ready. This is being used to poll endpoint status  of azure service connection